packages/cli/src/commands/pdf.js

```diff
@@ -4,7 +4,27 @@ import fs from 'fs-extra'
 import libPdf from '#lib/pdf/index.js'
 import open from 'open'
 import path from 'node:path'
-// import testcwd from '#helpers/test-cwd.js'
+import yaml from 'js-yaml'
+
+// FIXME: All this config loading might be better moved to another lib helper?
+// FIXME: Also it should probably load the same config validator(s) used in the build proc.
+/**
+ * @function loadConfig(path) - loads and validates quire config, returns an empty object if not found
+ * @param {path} string - file path to config file
+ */
```

Comment:
Makes sense -- amended

Reply:
Ah--oops, `loadConfig` can't throw because it runs at CLI init time and `quire new` has no config.yaml. Instead I have it `process.exit(1)` within `action()` if there is no configuration